documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -170,12 +165,12 @@ Enable or disable a read-only filesystem.
 
 ===== Fan
 
-Set the behaviour of a GPIO connected fan.
+Customise the behaviour of a GPIO connected fan.
```

Comment:
Should "GPIO connected fan" be ""GPIO-connected fan" ? Might be worth clarifying that (AFAIK) this is only for use by https://www.raspberrypi.com/products/raspberry-pi-4-case-fan/

Reply:
Ahhh excellent point, looks like the Active Cooler and Pi 5 case fan use a separate fan header. Thanks!